src/pages/[userId]/prospects.js

```diff
@@ -90,6 +92,7 @@ export default function Prospects() {
         </Grid>
       </Grid>
       {isLoadingProspects && <h1>Loading data...</h1>}
+      {prospectsError && <h1>Error loading prospects...</h1>}
       {prospects
```

Comment:
I think another thing we'll want here is that if there are no prospects, let's have a message about it so it's not just a blank screen.